src/shared/components/dateRangePicker/NewDatePicker.tsx

```diff
@@ -120,14 +146,6 @@ const DatePickerMenu: FC<Props> = ({onCollapse, timeRange, timeRangeLabel}) => {
 
   const handleSelectDate = (dates): void => {
     const [start, end] = dates
-    if (timeZone === 'UTC') {
-      if (start) {
-        start.setMinutes(start.getMinutes() + start.getTimezoneOffset())
-      }
-      if (end) {
-        end.setMinutes(end.getMinutes() + end.getTimezoneOffset())
-      }
-    }
```

Comment:
No need to adjust time with the time zone offset since the new date picker set the time from 00:00 to 23:59 regardless of the time zone. See the code here https://github.com/influxdata/ui/blob/c5ebfaf890fa29c0ccc37f9d220d35abda686f83/src/shared/components/dateRangePicker/NewDatePicker.tsx#L135-L164